lib/tasks/instrumentation_generator/instrumentation.thor

```diff
@@ -85,18 +87,35 @@ class Instrumentation < Thor
     )
   end
 
-  def config_block(library)
+  def append_to_newrelic_yml(name)
+    insert_into_file(
+      NEWRELIC_YML_LOCATION,
+      yaml_block(name),
+      after: "# instrumentation.bunny: auto\n"
+    )
+  end
+
+  def config_block(name)
```

Comment:
For both this method and `yaml_block`, I suppose their contents aren't blocks, but heredocs. Should the method names reflect that?

Reply:
I like what you have now. It gives you a block of configuration, and I don't get that "block" meaning confused with a Ruby callable block or a block or code or whatnot.